scss/_custom-forms.scss

```diff
@@ -149,7 +149,7 @@
   &:focus {
     border-color: $custom-select-focus-border-color;
     outline: 0;
-    @include box-shadow($custom-select-focus-box-shadow);
+    box-shadow: $custom-select-focus-box-shadow;
```

Comment:
Is this intentional?

Reply:
Yes, we want the focus state to always be there regardless of the global shadow variable. This matches our other inputs.